pkg/apiserver/interfaces/api/cloudshell.go

```diff
@@ -61,6 +61,14 @@ func (c *CloudShellAPIInterface) GetWebServiceRoute() *restful.WebService {
 		Returns(400, "Bad Request", bcode.Bcode{}).
 		Writes(apis.CloudShellPrepareResponse{}).Do(returns200, returns500))
 
+	ws.Route(ws.DELETE("/").To(c.destroyCloudShell).
```

Comment:
who will invoke this API to detroy？

Reply:
Users could destroy the cloud shell environment and recreate it, this could get the new permissions.